src/physical-and-chemical-data/data/inorganic-compounds.ts

```diff
@@ -509,4 +509,410 @@ export const InorganicCompounds: Record<string, PhysicalProperties> = {
+    formula: 'AlF3.3NaF',
+    formulaWeight: 209.94,
+    color: { nature: Colors.White },
+    crystallineForms: {
+      nature: CrystallineForms.Monoclinic,
+    },
+    refractiveIndex: { value: 1.3389 },
+    specificGravity: {
+      value: 2.9,
+      reference: SpecificGravityReference.Water,
+    },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Soluble,
+        adjective: Adjectives.Slightly,
+      },
+      HCl: {
+        nature: SolubilityNature.Insoluble,
+      },
+    },
+  },
+
+  AluminumSodiumSilicate: {
+    name: 'Aluminum sodium silicate',
+    formula: 'Al2O3.Na2O.6SiO2',
+    formulaWeight: 524.44,
+    color: { nature: Colors.Colorless },
+    crystallineForms: {
+      nature: CrystallineForms.Trigonal,
+    },
+    refractiveIndex: { value: 1.529 },
+    specificGravity: {
+      value: 2.61,
+      reference: SpecificGravityReference.Water,
+    },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Insoluble,
+      },
+      hotWater: {
+        nature: SolubilityNature.Insoluble,
+      },
+      acid: {
+        nature: SolubilityNature.Decomposes,
+      },
+    },
+  },
+
+  AluminumSulfate: {
+    name: 'Aluminum sulfate',
+    formula: 'Al2(SO4)3',
+    formulaWeight: 342.15,
+    color: { nature: Colors.White },
+    crystallineForms: {
+      nature: CrystallineForms.Crystalline,
+    },
+    refractiveIndex: { value: 1.529 },
+    specificGravity: {
+      value: 2.71,
+      reference: SpecificGravityReference.Water,
+    },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Soluble,
+        value: 31.3,
+        temperature: {
+          value: 0,
+          units: 'C',
+        },
+      },
+      hotWater: {
+        nature: SolubilityNature.Soluble,
+        value: 89,
+        temperature: {
+          value: 100,
+          units: 'C',
+        },
+      },
+    },
+  },
+
+  Alum: {
+    name: 'Alum, ammonium (tschermigite)',
+    formula: 'Al2(SO4)3.(NH4)2SO4.',
+    formulaWeight: 906.66,
+    color: { nature: Colors.Colorless },
+    crystallineForms: {
+      nature: CrystallineForms.Octahedral,
+    },
+    refractiveIndex: { value: 1.4594 },
+    specificGravity: {
+      value: 1.64,
+      substanceTemperature: {
+        value: 20,
+        units: 'C',
+      },
+      reference: SpecificGravityReference.Water,
+      referenceTemperature: {
+        value: 4,
+        units: 'C',
+      },
+    },
+    meltingOutcomes: [],
+    boilingOutcomes: [],
+    solubilitySheet: {
+      coldWater: {
+        nature: SolubilityNature.Soluble,
+        value: 3.9,
+        temperature: {
+          value: 0,
+          units: 'C',
+        },
+      },
+      hotWater: {
+        nature: SolubilityNature.Soluble,
+        value: Infinity,
```

Comment:
```suggestion
        value: Number.Infinity,
```

Reply:
Number.Infinity is throwing error `any
Property 'Infinity' does not exist on type 'NumberConstructor'`

Reply:
Current is okay then